doc/source/cice_index.rst

```diff
@@ -252,6 +252,10 @@ either Celsius or Kelvin units).
    "fswint", "shortwave absorbed in ice interior", "W/m\ :math:`^2`"
    "fswpenl", "shortwave penetrating through ice layers", "W/m\ :math:`^2`"
    "fswthru", "shortwave penetrating to ocean", "W/m\ :math:`^2`"
+   "fswthru_vdr", "vis dir shortwave penetrating to ocean", "W/m\ :math:`^2`"
```

Comment:
I think for these we should spell out these shortened versions just for clarity. I should also make sure this happens in the Icepack documentation. 
vis dir = visible direct, vis dif = visible diffuse, etc. These are obvious to us, but I don't think they are necessarily obvious to others.

Reply:
Part of the reason I used the abbreviations here is so that the table did not run over. Here is the icepack table:

https://cice-consortium-icepack.readthedocs.io/en/master/icepack_index.html

I can add the full names if people feel strongly about this.

Reply:
The longer spelled out forcing would be more consistent with the albedo components and I think more clear:
alndf | albedo: near IR, diffuse 
alndr | albedo: near IR, direct 
alvdf | albedo: visible, diffuse
alvdr | albedo: visible, direct

If they run over it should just wrap (see dardg1(n)dt).

Reply:
Ok. Made the suggested change to CICE. I have another Icepack PR coming in the next couple days where I will fix it there.

Reply:
Thumbs up